### PR TITLE
Fixed ci github actions to make it bors compatible again

### PR DESCRIPTION
**Pull request by hgrecco**
- [ ] Closes # (insert issue number)
- [ ] Executed ``pre-commit run --all-files`` with no errors
- [ ] The change is fully covered by automated unit tests
- [ ] Documented in docs/ as appropriate
- [x] Added an entry to the CHANGES file

### CODE REVIEW

**Comment by jules-ch** [comment]
Looks good, do you want to merge as is?
Failing CI on documentation has been fixed in 552e02de98a61da50845bfb19090087bf4e93583

Rebase or merge as is, it won't be a problem anyway.

File: CHANGES [hunk 1]
```diff
@@ -4,6 +4,7 @@ Pint Changelog
 0.17 (unreleased)
```
Comment (jules-ch) [inline CHANGES:4]: 0.17 has been released can you rebase ?

Comment (hgrecco) [reply CHANGES]: I merged the master branch into my develop. Should do the trick

**Comment by hgrecco** [comment]
Is this ready to merge?

**Comment by keewis** [comment]
no, you still need to move the changelog entry to the `0.18` section

**Comment by hgrecco** [comment]
I rebased and fixed an issue with bors that prevented using it.

**Comment by hgrecco** [comment]
I will remove the universal in the next.

**Comment by hgrecco** [comment]
@jules-ch bors is now fixed for pint

**Pull request merged**